tests/Makefile

```diff
@@ -58,4 +62,5 @@ check: unit \
 	stop_condition_never_filter_out \
 	stop_condition_present \
 	stop_condition_length \
-	stop_condition_exist
+	stop_condition_exist \
+	integration_tests
```

Comment:
Should this be called integration tests? Maybe something like complex_type_tests?

Reply:
My intention is to add more tests, where model is generated from actual taste projects,

Reply:
Closing, to be done in a separate MR